README.md

```diff
@@ -113,16 +132,42 @@ docker run \
   --rm \
   --interactive \
   --tty \
-  --volume $(pwd)/output.csv:/tmp/output.csv \
-  --volume $(pwd)/queries:/tmp/queries \
+  --volume $(pwd)/output.csv:/output.csv \
+  --volume $(pwd)/queries:/queries \
   comunica/sparql-benchmark-runner \
   --endpoint https://dbpedia.org/sparql \
-  --queries /tmp/queries \
-  --output /tmp/output.csv \
+  --queries /queries \
+  --output /output.csv \
   --replication 5 \
   --warmup 1
 ```
+[example queries](https://gist.github.com/davidsbatista/cdce57196bf84e3a988427b4d9ef9035) in `./queries/C1.text` from
```

Comment:
I can delete it, I just wanted to have some queries associated with the docker example since it precise an real endpoint. I did not saw a whatdiv SPARQL endpoint online, maybe using that as an example would be better because then the queries presented earlier could be used.

Reply:
I don't think this is needed, as people that want to use a SPARQL benchmark runner will be knowledgeable enough about SPARQL queries and endpoints.